server/libs/userlibs.js

```diff
@@ -0,0 +1,52 @@
+const {User} = require('../DataBase/models/user');
```

Comment:
import instead of require

Reply:
Node.js doesn't support ES6's import yet. So that's why used require